charts/prefect-server/values.schema.json

```diff
@@ -150,7 +156,8 @@
         "loggingLevel": {
           "type": "string",
           "title": "Logging Level",
-          "description": "sets PREFECT_LOGGING_SERVER_LEVEL"
+          "description": "sets PREFECT_LOGGING_SERVER_LEVEL",
+          "enum": ["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"]
```

Comment:
based on the docs, i've added this as a check to ensure we're passing only correct values/